src/io/swiftsim_io.cpp

```diff
@@ -163,7 +163,7 @@ void SwiftSimReader_t::ReadHeader(int ifile, SwiftSimHeader_t &header)
   /* Read group ID used to indicate that a particle is in no FoF group */
   string buf;
   ReadAttribute(file, "Parameters", "FOF:group_id_default", buf);
-  Header.NullGroupId = std::stoi(buf);
+  Header.NullGroupId = (HBTInt)std::stoll(buf);
```

Comment:
@jchelly I changed the dtype of `NullGroupId` to `HBTInt` to allow it to be a long long in the future. It compiles and reads the value correctly. Do you have any thoughts/comments about this change?